src/main/java/Deadline.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Representation of a Deadline task. Inherits from Task.
+ */
+public class Deadline extends Task {
+    String deadLine;
+    LocalDate localDate;
+    LocalTime localTime;
+
+    /**
+     * Class constructor. Creates a not done Deadline object with name set to the specified taskName
+     * and deadline set to the specified deadLine. Converts and stores the deadline as LocalDate and LocalTime.
+     * @param taskName The specified name of the new Deadline object.
+     * @param deadLine The specified deadline of the new Deadline object.
+     */
+    public Deadline(String taskName, String deadLine) {
+        super(taskName);
+        this.deadLine = deadLine;
+
+        String[] deadLineSplit = deadLine.split(" ");
```

Comment:
perhaps for the array to represent a collection of details of this deadline, name [descriptions](url) in plural form is better?